core/src/tx.rs

```diff
@@ -195,8 +195,6 @@ impl AcceptedTransaction {
         }
 
         for instruction in &self.payload.instructions {
-            let account_id = self.payload.account_id.clone();
```

Comment:
did you get this one from the clipppy now?

Reply:
Clippy only wanted me to rename the variable. I noticed that all it was doing was `inc/dec` the refcount.